p/scripts/extra.js

```diff
@@ -123,6 +123,29 @@ function init_password_observers() {
 	});
 }
 
+// overwrites the href attribute from the url input
+function updateHref() {
+	const urlField = document.getElementById(this.getAttribute('data-input'));
+	const url = urlField.value;
+	if (url.length > 0) {
+		this.href = url;
+		return true;
+	} else {
+		urlField.focus();
+		this.removeAttribute('href');
+		return false;
+	}
+}
+
+// set event listener on "show url" buttons
```

Comment:
This seems a bit complex, although presumably it would be possible to replace all of them with `focus` or `click`. (If not, I suspect you need focus in addition to the rest.)

It seems more intuitive to use the `input` event on the text/url input.

Reply:
mousedown and keydown would be enough in general. The mouseover is used to update the link preview.

(see browser corner)
![grafik](https://user-images.githubusercontent.com/1645099/155861095-18d5ef04-f843-4c60-ac05-e91203ddba96.png)

I do not understand why it would be better to use the input event

Reply:
I changed to click event.